ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/regression/LinearRegression.java

```diff
@@ -58,7 +58,7 @@ public class LinearRegression implements Trainable, Predictable {
     //RMSProp
     private static final double DEFAULT_DECAY_RATE = 0.9;
 
-    private static final int DEFAULT_EPOCHS = 10;
+    private static final int DEFAULT_EPOCHS = 1000;
     private static final int DEFAULT_INTERVAL = -1;
     private static final int DEFAULT_BATCH_SIZE = 1;
```

Comment:
Can you help fix the hard coded issue in line 203
```
LinearSGDTrainer linearSGDTrainer = new LinearSGDTrainer(objective, optimiser, epochs, DEFAULT_INTERVAL, DEFAULT_BATCH_SIZE, seed);
```

Reply:
Sure, do you mean DEFAULT_INTERVAL, DEFAULT_BATCH_SIZE, seed these 3 hard coded ones?

Reply:
yes